packages/flutter/lib/src/material/flat_button.dart

```diff
@@ -94,27 +98,34 @@ class FlatButton extends StatelessWidget {
   /// ```
   final Color color;
 
-  /// The primary color of the button when the button is in the down (pressed) state.
+  /// The primary color of the button when the button is in the down (pressed)
+  /// state.
+  ///
   /// The splash is represented as a circular overlay that appears above the
-  /// [highlightColor] overlay. The splash overlay has a center point that matches
-  /// the hit point of the user touch event. The splash overlay will expand to
-  /// fill the button area if the touch is held for long enough time. If the splash
-  /// color has transparency then the highlight and button color will show through.
+  /// [highlightColor] overlay. The splash overlay has a center point that
+  /// matches the hit point of the user touch event. The splash overlay will
+  /// expand to fill the button area if the touch is held for long enough time.
+  /// If the splash color has transparency then the highlight and button color
+  /// will show through.
```

Comment:
I recommend avoiding rewrapping text unless it's particularly egregious (so e.g. your change above where you're changing a 100+ character line into two lines is fine, but here it doesn't really help anything).

Reply:
(mostly just because it makes reviewing the patch harder, and in practice we're all likely to have slightly different wrapping algorithms in our IDEs so we're just going to keep going back and forth if we each do this...)

Reply:
Well, OK, but let's have a conversation sometime about having a style guide that can be applied algorithmically (and so consistently) rather than subjectively. Just like double-spaces before a period bother you, having doc comment lines that go over the line limit bother me... :-)

Reply:
Oh I'm not saying we shouldn't rewrap the lines, that's fine. I'm just saying we shouldn't rewrap them in otherwise unrelated PRs, because it makes the reviews harder.

If you want to submit a PR that rewraps all the comments appropriately then that would be fine.

Reply:
(Readability is subjective, though, and the style guide is optimising for readability over all else, so it'll never be truly algorithmic.)